onwuvic/MyDiary
# #159813683 Integrate get all diary endpoint with frontend UI page

#### What does this PR do?
Enable user to see all the diaries they have created on the frontend diaries page

#### Description of Task to be completed?
- Integrate get all diaries API with diaries UI page

#### How should this be manually tested?
- visit `https://onwuvic.github.io/MyDiary/UI/` and login if have an account
- When directed to diaries page you will see all the diaries you have created or no diary if you don't one.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
- Story type: feature
- Story title: Consume get All diary API with es6 frontend
- Story ID: #159813683

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
